crest/Assets/Crest/Crest/Shaders/OceanInputs/Resources/AnimWavesGerstnerGeometry.shader

```diff
@@ -0,0 +1,158 @@
+﻿// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+// Generates waves from geometry that is rendered into the water simulation from a top down camera. Expects
+// following data on verts:
+//   - POSITION: Vert positions as normal.
+//   - TEXCOORD0: Axis - direction for waves to travel. "Forward vector" for waves.
+//   - TEXCOORD1: X - distance from nearest side, needed if "feather from spline ends" is non-zero.
+//                Y - 0 at start of waves, 1 at end of waves
+//
+//  uv1.x = 0 -------- uv1.x = 50m --------- uv1.x = 0
+//  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ uv1.y = 0             |
+//  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  |                    |  uv0 - wave direction vector
+//  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  |                   \|/
+//  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ uv1.y = 1
+//  ------------------- shoreline --------------------
+//
+
+Shader "Crest/Inputs/Animated Waves/Gerstner Geometry"
+{
+    Properties
+    {
+		// Controls ramp distance over which waves grow/fade as they move forwards
+		_FeatherWaveStart("Feather wave start (0-1)", Range( 0.0, 0.5 ) ) = 0.1
+		// Fade in waves from the sides
+		_FeatherFromSplineEnds("Feather from spline ends (m)", Range( 0.0, 100.0 ) ) = 0.0
+		// Can be set to 0 to make waves ignore shallow water
+		_RespectShallowWaterAttenuation("Respect Shallow Water Attenuation", Range(0, 1)) = 1
+	}
+
+    SubShader
+    {
+		// Additive blend everywhere
+		Blend One One
+		ZWrite Off
+		ZTest Always
+		Cull Off
+
+        Pass
+        {
+            CGPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+			#pragma enable_d3d11_debug_symbols
+
+            #include "UnityCG.cginc"
+
+			#include "../../OceanGlobals.hlsl"
```

Comment:
This file could use a format. Whitespace is inconsistent.

Reply:
i'm 4 attempts in now and each time it seems to get worse! i manually retabbing in vs, format doc in vs, checking in sublime, manually changing tabs to spaces in sublime. will think about how to tackle this

Reply:
Looks good with latest.